identityClient/api.go

```diff
@@ -1196,13 +1196,15 @@ type PAMJiraPlugin struct {
 	ID                   int64  `json:"id"`
 	AutomationAuthHeader string `json:"automation_auth_header"`
 	BotUserEmail         string `json:"bot_user_email"`
+	JiraURL              string `json:"jira_url"`
```

Comment:
JiraURl is very abstract / hiding some implicit context you have now....suggest something more explicit like `JiraHostURL` or `JiraAPIURL`